packages-exp/firebase-exp/rollup.config.js

```diff
@@ -129,6 +129,38 @@ const componentBuilds = pkg.components
     // It is needed for handling sub modules, for example firestore/lite which should produce firebase-firestore-lite.js
     // Otherwise, we will create a directory with '/' in the name.
     const componentName = component.replace('/', '-');
+
+    if (component === 'messaging') {
+      return [
+        {
+          input: `${component}/index.cdn.ts`,
+          output: [
+            {
+              file: resolve(component, pkg.main),
+              format: 'cjs',
+              sourcemap: true
+            },
+            {
+              file: resolve(component, pkg.module),
+              format: 'es',
+              sourcemap: true
+            }
+          ],
+          plugins: [...plugins, typescriptPlugin],
+          external
+        },
+        {
+          input: `${component}/index.cdn.ts`,
+          output: createUmdOutputConfig(
+            `firebase-${componentName}.js`,
```

Comment:
We need to build a cdn script for both the client code and the sw code, with `firebase-messaging.js` and `firebase-messaging-sw.js` as the name respectively.

Reply:
Applies to the release build config too.